CODE_OF_CONDUCT.md

```diff
@@ -1,3 +1,9 @@
+<!--
+Copyright (c) 2023, Cisco and/or its affiliates.
+All rights reserved.
+See LICENSE file in this distribution.
+SPDX-License-Identifier: Apache-2.0
+-->
```

Comment:
XML comment for .md?  Not sure this is correct (same directly below).

Reply:
Markdown allows inline HTML. 

https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#hiding-content-with-comments